src/programs/unblur/unblur.cpp

```diff
@@ -296,17 +305,12 @@ bool UnBlurApp::DoCalculation()
 
 	//my_current_job.PrintAllArguments();
 
-	// Profiling
-	wxDateTime	overall_start = wxDateTime::Now();
-	wxDateTime 	overall_finish;
-	wxDateTime 	read_frames_start;
-	wxDateTime	read_frames_finish;
-	wxDateTime	first_alignment_start;
-	wxDateTime	first_alignment_finish;
-	wxDateTime	main_alignment_start;
-	wxDateTime	main_alignment_finish;
-	wxDateTime	final_alignment_start;
-	wxDateTime	final_alignment_finish;
+	// StopWatch objects
+	cistem_timer::StopWatch unblur_timing;
+	StopWatch profile_timing;
+	StopWatch profile_timing_refinement_method;
+
```

Comment:
Do I understand it right that the unblur_timer is always active because you explicitly define the namespace, while the two other will only be active when `HYPOTHETICAL_CONTROL_OVER_TIMER` is defined?